governance-art-collection.md

```diff
@@ -0,0 +1,15 @@
+| Title               | Governance Art Collection |
+| Author              | OfficialCryptos |
+| Featured text       | FIXME |
```

Comment:
Featured text?
Something like:
"Decred got the art of governance down"

Reply:
I like the featured text idea by Phoenix, but i'll leave two more:

1- Witness the evolution of decentralization as Decred meticulously crafts its governance DNA

2- Unravel the DNA of decentralization with Decred's pioneering governance model